serializations.md

```diff
@@ -631,16 +653,18 @@ The channel off-chain transaction is not included directly in the transaction tr
 
 #### Channel
 ```
-[ <initiator>        :: id()
-, <responder>        :: id()
-, <total_amount>     :: int()
-, <initiator_amount> :: int()
-, <channel_reserve>  :: int()
-, <delegates>        :: [id()]
-, <state_hash>       :: binary()
-, <round>            :: int()
-, <lock_period>      :: int()
-, <closes_at>        :: int()
+[ <initiator>           :: id()
+, <responder>           :: id()
+, <channel_amount>      :: int()
+, <initiator_amount>    :: int()
+, <responder_amount>    :: int()
+, <channel_reserve>     :: int()
+, <delegates>           :: [id()]
+, <state_hash>          :: binary()
+, <round>               :: int()
+, <lock_period>         :: int()
+, <closes_at>           :: int()
+, <force_blocked_till>  :: int()
```

Comment:
Remember to change this after merging force-progress PR (or, since there is no force progress until it's merged, why not change now?)